browser/src/canvas/sections/AutoFillMarkerSection.ts

```diff
@@ -1,3 +1,5 @@
+/** */
```

Comment:
Yes, please see commit message here:
* https://github.com/CollaboraOnline/online/pull/9442/commits/9d55c6aa3ba4b63ca3239cfa05ca6e069c952287

There can be better solutions of course. Like filling that with a more sensible string etc but this is also nor harm and works.

Reply:
Or we can add a new line after creating the source-map string for files. That'd also be an option i guess. This is the fastest working solution, since there are many changes, i needed this to work.

Reply:
@gokaysatir can we wait a week and then replace these with some helpful top/bottom comments - we should write a script that adds our stock JS / emacs header at the top, and our stock sensible VIM header at the bottom I think - to get tabs and spaces right etc. - thanks ! =)